ui/app/routes/vault/cluster/policies/policy.js

```diff
@@ -0,0 +1,23 @@
+import UnloadModelRecord from 'vault/routes/vault/cluster/unload-model-record';
+import { inject as service } from '@ember/service';
+
+export default class PolicyRoute extends UnloadModelRecord {
+  @service store;
+
+  beforeModel() {
+    const params = this.paramsFor(this.routeName);
+    let policyType = this.policyType();
+    if (policyType === 'acl' && params.policy_name === 'root') {
+      return this.transitionTo('vault.cluster.policies', 'acl');
+    }
+  }
+
```

Comment:
this file previously had a setupController method, but I don't think we need it anymore?

```js
 setupController(controller, model) {
    controller.setProperties({
      model: model.policy,
      capabilities: model.capabilities,
      policyType: this.policyType(),
    });
  }